src/endpoints/chats.js

```diff
@@ -785,9 +785,9 @@ router.post('/search', validateAvatarUrlMiddleware, function (request, response)
 
             // Search through title and messages of the chat
             const fragments = query.trim().toLowerCase().split(/\s+/).filter(x => x);
-            const text = [chatFile.path.split(/[\\/]/).pop().replace(/.jsonl$/, ''),
-                ...messages.map(message => message?.mes)].join('\n').toLowerCase();
-            const hasMatch = fragments.every(fragment => text.includes(fragment));
+            const text = [path.parse(chatFile.path).name,
+                ...messages.map(message => message?.mes)].join("\n").toLowerCase();
```

Comment:
Oh, right, the linter…
```suggestion
                ...messages.map(message => message?.mes)].join('\n').toLowerCase();
```